lib/instructions/instruction_names.c

```diff
@@ -0,0 +1,1060 @@
+/* EVMC: Ethereum Client-VM Connector API.
+ * Copyright 2018 Pawel Bylica.
+ * Licensed under the MIT License. See the LICENSE file.
+ */
+
+#include <evmc/instructions.h>
+
+static const char* constantinople_names[256] = {
```

Comment:
These lists are hard to maintain.
Maybe at least add delimiting comment lines like
```
   // 0x0
   "STOP",
   "ADD",
...
   // 0x10
...
```

Reply:
Added numbers in comments.